x-pack/plugins/security/server/routes/api/v1/privileges.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { buildPrivilegeMap } from '../../../lib/privileges/privileges';
+
+export function initPrivilegesApi(server) {
+  const config = server.config();
+  const kibanaVersion = config.get('pkg.version');
+  const application = config.get('xpack.security.rbac.application');
+  const savedObjectTypes = server.savedObjects.types;
+
+  server.route({
+    method: 'GET',
+    path: '/api/security/v1/privileges',
```

Comment:
We should have api integration tests for this endpoint.

Why do you need the `v1` thing? I'd like to avoid ad hoc versioning across individual endpoints if we can.

Reply:
All of the security endpoints use the `v1` path component, we were just following suit here, not something I'm particularly fond of either.

Reply:
I added API integration tests for this endpoint as well.